src/analyser/InternPool.zig

```diff
@@ -4532,19 +4532,22 @@ test "function type" {
         },
     } });
 
-    const @"fn() align(4) callconv(.C) type" = try ip.get(gpa, .{ .function_type = .{
+    const default_ccc_name = @tagName(std.builtin.CallingConvention.c); // We need to name match a dynamic value
+    const expected = try std.fmt.allocPrint(gpa, "fn() align(4) callconv(.{s}) type", .{default_ccc_name});
+    defer gpa.free(expected);
```

Comment:
```suggestion
    const expected = try std.fmt.comptimePrint("fn() align(4) callconv(.{s}) type", .{default_ccc_name});
```

Reply:
Neat !

(Didn't apply it directly because of the `try`)